src/components/Permissions/DataList.jsx

```diff
@@ -1,15 +1,117 @@
+/* eslint-disable no-unsafe-optional-chaining */
 import React from 'react'
+import { Link } from 'react-router-dom'
 import Page from 'components/Page'
 import PageTitle from 'components/PageTitle'
-import { useI18n } from 'cozy-ui/transpiled/react/I18n'
+import Typography from 'cozy-ui/transpiled/react/Typography'
+import List from 'cozy-ui/transpiled/react/MuiCozyTheme/List'
+import ListItem from 'cozy-ui/transpiled/react/MuiCozyTheme/ListItem'
+import ListItemText from 'cozy-ui/transpiled/react/ListItemText'
+import ListItemIcon, {
+  mediumSize
+} from 'cozy-ui/transpiled/react/MuiCozyTheme/ListItemIcon'
+import Divider from 'cozy-ui/transpiled/react/MuiCozyTheme/Divider'
+import Icon from 'cozy-ui/transpiled/react/Icon'
+import Spinner from 'cozy-ui/transpiled/react/Spinner'
+import { APPS_DOCTYPE, KONNECTORS_DOCTYPE } from 'doctypes'
+import withAllLocales from '../../lib/withAllLocales'
+import { getPermissionIconName } from '../../containers/helpers/permissionsHelper'
```

Comment:
```suggestion
import { getPermissionIconName } from '/containers/helpers/permissionsHelper'
```

Reply:
we prefer to use webpack alias in order to be less fragile when moving components in different folder.

Could you also add a guideline about it, 🙏 ?

Reply:
`'src/containers/helpers/permissionsHelper'`

Reply:
/containers 🙏